src/webui/www/private/scripts/client.js

```diff
@@ -37,6 +37,7 @@ var updateMainData = function () {};
 var alternativeSpeedLimits = false;
 var queueing_enabled = true;
 var syncMainDataTimerPeriod = 1500;
+var torrentsTableSelectedRows = [];
```

Comment:
Why did you declare this var here? Why not in onSuccess, where it is used?

Reply:
I think it's from a previous iteration of this PR where the variable was used elsewhere. I've moved its declaration to `onSuccess`.

Reply:
> I've moved its declaration to onSuccess.

Ok. Squash this commit into previous one.

Reply:
All squashed.